virttest/utils_misc.py

```diff
@@ -2639,12 +2639,27 @@ def get_free_disk(session, mount):
     return int(free)
 
 
+def get_win_mon_free_mem(session):
+    """
+    Get Performace Monitored Free memory for Windows.
+
+    :param session: shell Object.
+
+    :return string: freespace M-bytes
+    """
+    cmd = 'typeperf "\Memory\Free & Zero Page List Bytes" -sc 1'
+    output = session.cmd_output(cmd)
```

Comment:
@balamuruhans 
   "typeperf" is an internal cmd of windows, so we don't need to install it manually.